distributed/scheduler.py

```diff
@@ -2215,9 +2215,11 @@ async def add_worker(
                 except Exception as e:
                     logger.exception(e)
 
+            recommendations: dict
             if nbytes:
                 for key in nbytes:
-                    ts: TaskState = self.tasks.get(key)
+                    tasks: dict = self.tasks
+                    ts: TaskState = tasks.get(key)
```

Comment:
Can we not type `self.tasks` at the class level?  Or does this need the `@cclass` treatment to work?

Reply:
Yeah it needs the `@cclass` treatment.